### ISSUE TITLE
Select precise font instead of a font family 

### ISSUE THREAD

**Issue by GlennMiller** [warpdotdev/Warp#1377]
### Describe the solution you'd like?

```text
I prefer the JetBrains Mono Thin font in my iTerm2, VSCode, et al.

In Warp, I can only select JetBrains Mono font. 

Could you consider this, or am I just missing something?
```

**Comment by iquad** [comment]
+1 here.

**Comment by elviskahoro** [comment]
Thanks for opening. Will report back with updates when tere are any

**Comment by iquad** [comment]
so, anything @elviskahoro ?

**Comment by warpdotdev-dx** [comment]
Still no progress @mert


— elvis kahoro

**Comment by saligrama** [comment]
Any update on this? I would like to use a custom build of Iosevka selecting the extended width variant -- currently, I can only select the standard width.

**Comment by pure-blue** [comment]
I would also really like this...

**Comment by srdjadzogaz** [comment]
+1 for this.

**Comment by pure-blue** [comment]
> +1 for this. I would also like to use semi-bold Iosevka as default one is too thin.

Yeah, implementation of this feature request would allow you to load every so-called "style" of a font (semi-bold, regular, light, thin, etc.) into your fonts library and then choose a style and switch between them, similar to iTerm2. IMHO this should be standard.

<img width="954" alt="Screenshot 2024-02-25 at 1 50 09 PM" src="https://github.com/warpdotdev/Warp/assets/79292724/342297c7-e364-4450-acc0-41a70def78f1">

**Comment by vtamm** [comment]
Wanted to give Warp another go today and was very surprised to discover that I was unable to pick the font style in the Appearance settings. I want to use Operator Mono SSm in my Warp terminal but the settings for some reason default to the bold version of the font which is less than ideal in a CLI, to say the least.

+1 please get this fixed if you want more people using your app! My colleague uses the same font as me for his terminals so he's not going to want to try out Warp either until this feature is in.

**Comment by merdemkoc** [comment]
Any update on this? We still can't select custom fonts from settings. Some installed fonts aren't available. It's mentioned in https://github.com/warpdotdev/Warp/issues/3715
